elastic/apm-server
# Add missing `agent.activation_method` mapping

## Motivation/summary


Add missing agent.activation_method mapping to `logs-apm.app`, `metrics-apm.app`, `logs-apm.error` and `metrics-apm.internal`

## Checklist




## How to test these changes


- Send events (metrics, logs, transactions, spans) with `agent.activation_method` metadata
- Check that the event is indexed properly with the mapped field `agent.activation_method`

## Related issues


#9622 
Fixes #10551
